WebServices/AzureNotificationHub/Metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
I know I told you to add a metadata file, but am I correct in thinking that the new sample gallery doesn't use it, as the metadata is now in the readme?

Reply:
You are correct but the new sample gallery isn't out yet. I'll be running a tool to batch convert these xml files to yaml headers for the new system when it's read. I spoke briefly with Craig and we both felt it'd be better to just make this match existing pattern so it's processed in the same way.